src/Core/CustomObjects/Config.cs

```diff
@@ -6,12 +6,14 @@
 using System.IO;
 using System.Linq;
 
-namespace Axe.Windows.Desktop.UIAutomation.CustomObjects
+namespace Axe.Windows.Core.CustomObjects
 {
-    class Config
+    public class Config
     {
         [JsonProperty("properties")]
+#pragma warning disable CA1819 // Properties should not return arrays: represents a JSON collection
```

Comment:
Possibly move line 14 up above line 13, just to keep the property neatly tied to the JsonProperty? (Compiler is fine either way)